README.md

```diff
@@ -33,10 +36,14 @@ This repository contains System Design resources which are useful to learn conce
 - [API Gateway](https://www.nginx.com/learn/api-gateway/)
 - [Distributed Locking](https://martin.kleppmann.com/2016/02/08/how-to-do-distributed-locking.html)
 - [Checksum](https://www.lifewire.com/what-does-checksum-mean-2625825)
+- [Micro Frontends](https://newsletter.systemdesign.one/p/micro-frontends)
+- [Consistent Hashing](https://newsletter.systemdesign.one/p/what-is-consistent-hashing)
+- [Scalability 101 by WhatsApp](https://newsletter.systemdesign.one/p/whatsapp-engineering)
```

Comment:
This one's a "listicle". Addresses the social aspects of development instead of the technical aspects.

May not be relevant.